spa/assets/object/ObjectPreview.htm

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>

```

Comment:
I will look into this. Originally when I tried to put a way to view x_ite inside a vue component (AV viewer into the info modal), it would completely break 3d for the rest of ctr because it was trying to load 2 instances of x_ite into single browser window. This would cause whatever I loaded into the modal to be loaded into the modal and the world if the user was in 2d, or just not load at all if the user was in 3d. However, since I am using a new window for the object properties rather than a modal, this may not be an issue any longer.